locales/es_ES/pcsx2_Main.po

```diff
@@ -445,7 +444,7 @@ msgstr "Muy agresivo"
 
 #: pcsx2/gui/AppConfig.cpp:981
 msgid "Mostly Harmful"
-msgstr "Dañino"
+msgstr "Perjudicial"
```

Comment:
English is Mostly Harmful.
Doesnt it need Mayormente?

Reply:
You are right, that does need something that separates "Harmful" from "Mostly Harmful". Correcting.